runtime/near-vm-runner-standalone/Cargo.toml

```diff
@@ -39,7 +39,16 @@ near-test-contracts = { path = "../near-test-contracts" }
 [features]
 default = []
 no_cache = ["near-vm-runner/no_cache"]
-protocol_feature_alt_bn128 = ["near-vm-logic/protocol_feature_alt_bn128", "near-vm-runner/protocol_feature_alt_bn128"]
+protocol_feature_alt_bn128 = ["near-primitives/protocol_feature_alt_bn128", "near-vm-logic/protocol_feature_alt_bn128", "near-vm-runner/protocol_feature_alt_bn128"]
+protocol_feature_evm = ["near-primitives/protocol_feature_evm"]
+protocol_feature_block_header_v3 = ["near-primitives/protocol_feature_block_header_v3"]
+protocol_feature_add_account_versions = ["near-primitives/protocol_feature_add_account_versions"]
+protocol_feature_tx_size_limit = ["near-primitives/protocol_feature_tx_size_limit"]
+protocol_feature_fix_storage_usage = ["near-primitives/protocol_feature_fix_storage_usage"]
+nightly_protocol_features = ["nightly_protocol", "near-primitives/nightly_protocol_features", "protocol_feature_evm", "protocol_feature_block_header_v3", "protocol_feature_alt_bn128", "protocol_feature_add_account_versions", "protocol_feature_tx_size_limit", "protocol_feature_fix_storage_usage", "protocol_feature_restore_receipts_after_fix", "protocol_feature_cap_max_gas_price"]
+protocol_feature_restore_receipts_after_fix = ["near-primitives/protocol_feature_restore_receipts_after_fix"]
+protocol_feature_cap_max_gas_price = ["near-primitives/protocol_feature_cap_max_gas_price"]
+nightly_protocol = ["near-primitives/nightly_protocol"]
```

Comment:
why do we need all these features?

Reply:
While fixing CI checks, lines like these;
Before now, lines like these enabled features that prior to the decoupling, existed in the workspace.
https://github.com/near/nearcore/blob/928f15df25ca4128d88b98e930d580ed2023d576/Makefile#L37
Now those features are defined in `nearcore`. This crate, however, doesn't directly depend on `nearcore`. So we have to adapt the needed features to directly match the available dependencies.